components/status/StatusActionsMore.vue

```diff
@@ -149,8 +150,8 @@ function showFavoritedAndBoostedBy() {
 
           <CommonDropdownItem
             :text="status.favourited ? $t('action.favourited') : $t('action.favourite')"
-            :icon="status.favourited ? 'i-ri:heart-3-fill' : 'i-ri:heart-3-line'"
-            :class="status.favourited ? 'text-rose' : ''"
+            :icon="`${useStarFavoriteIcon ? 'i-ri:star-' : 'i-ri:heart-3-'}${status.favourited ? 'fill' : 'line'}`"
```

Comment:
You should try to always have the classes as fully written out and not dynamically calculated and concatinated because this results in issues

try something like

```ts
useStarFavoriteIcon
  ? status.favourited ? 'i-ri:star-fill'    : 'i-ri:star-line'
  : status.favourited ? 'i-ri:heart-3-fill' : 'i-ri:heart-3-line'
```

this also applies to lines below

Reply:
OK, this was fixed by a new commit. 👍🏻